core/templates/dev/head/pages/learner-dashboard-page/learner-dashboard-page.directive.html

```diff
@@ -739,7 +739,7 @@ <h1 class="stat-value"><[$ctrl.subscriptionsList.length]></h1>
                     </div>
                     <div>
                       <button class="btn btn-success feedback-thread-new-message-send"
-                              ng-click="$ctrl.addNewMessage(threadId, $ctrl.newMessage.text)"
+                              ng-click="$ctrl.addNewMessage($ctrl.threadId, $ctrl.newMessage.text)"
```

Comment:
@nithusha21, this line fixes issue #7201 but after that, I found another issue which appears after adding 2 messages in the thread (see the image in the main comment of the PR). I tried fixing the issue but I'm not very confident with the solution as I was not very comfortable with reading the existing code and understand the behavior. But I've tried to fix it and test it thoroughly. Can you please check?